src/main/resources/application.properties

```diff
@@ -1,6 +0,0 @@
-spring.datasource.url=
```

Comment:
Would like to leave this but with empty values. A dev can then easily see which properties need to be filled in

Reply:
Restored